src/celeritas/mat/MaterialParams.cc

```diff
@@ -63,10 +63,25 @@ std::shared_ptr<MaterialParams>
 MaterialParams::from_import(ImportData const& data)
 {
     CELER_EXPECT(!data.materials.empty());
+    CELER_EXPECT(!data.elements.empty());
 
-    // Create MaterialParams input for its constructor
     MaterialParams::Input input;
 
+    // Populate input.isotopes
+    for (auto const& isotope : data.isotopes)
+    {
+        MaterialParams::IsotopeInput isotope_params;
+        isotope_params.label = isotope.name;
+        isotope_params.atomic_number = AtomicNumber{isotope.atomic_number};
+        isotope_params.atomic_mass_number
+            = AtomicNumber{isotope.atomic_mass_number};
+        // Convert from MeV (Geant4) to MeV/c^2 (Celeritas)
+        isotope_params.nuclear_mass = units::MevMass{
+            isotope.nuclear_mass / std::pow(constants::c_light, 2)};
```

Comment:
I think there was some confusion about how this value is represented. I think you should just have `MevMass{nuclear_mass}` with no division by `c^2`. Check the value for carbon-12: `native_value_from(isotope.nuclear_mass) / celeritas::constants::atomic_mass` should be exactly 12.

Reply:
Ohh I misread the `MevMass = Quantity<MevPerCsq>;`. The unit _is_ `MeV/c^2` and thats equivalent to `MeV` in G4. Not that we are storing the value after dividing per c^2. I'll fix that.

> `native_value_from(isotope.nuclear_mass) / celeritas::constants::atomic_mass` should be exactly 12

Yes! Agreed.

Now, this point made me double check the `G4NucleiProperties::GetNuclearMass(...)`, which has 2 overloads:
```cpp
GetNuclearMass(G4double A, G4double Z);
GetNuclearMass(G4int A, G4int Z);
```
Which are slightly different. The overload with doubles is more precise, so I should use that one. @whokion do you also agree with the need to use the one with doubles as inputs?

Reply:
Yes, @stognini  are right and should use `GetNuclearMass(G4double A, G4double Z)` as it also covers `GetNuclearMass(G4int A, G4int Z)`, (i.e., using the known PDF mass when A is close to an integer, otherwise, taking into account an estimated mass access and binding energy).

Reply:
@stognini Looking at the source code, I think you want to use the `int` function signature because that uses a table lookup for the data rather than a rough functional approximation.

Reply:
Z and A shouldn't ever be doubles. It looks like the `int, int` method falls back to the functional approximation if it's not in any of the tables.

The whole "z as a double" reminds me of `if (Z < 1.5)`... 😠

Reply:
Oh man, you're right. I missed the `NuclearMass(...)` at the bottom when skimming through it...

Yeah, G4 accessors are weird. For `A` you only get a double that represents the effective number of nucleons, which is pretty much the same as giving you its amu, so that's why it returns a double. For `Z` it should be `int` for sure.

Reply:
Okay, I was wrong and agree that we 'd better use `GetNuclearMass(G4int A, G4int Z)` for storing the nuclear mass for the isotope case (as the result is same anyway even if GetNuclearMass(G4double A, G4double Z) is used with int A and int Z.  The user case for GetNuclearMass(G4double A, G4double Z) may be only for using the effective (or weighted) atomic mass number of an element, which is irrelevant for this importing.  Also wondering what real use cases are for using GetNuclearMass(G4double A, G4double Z) in any application, in which may need to use this on-the-fly calculation.